ztag/schema.py

```diff
@@ -532,6 +532,31 @@ def ztag_zgrab2_transformed(service, results):
                              "the handshake.")
 }))
 
+ztag_mongodb = ztag_zgrab2_transformed(service="MongoDB", results=SubRecord({
+        "build_info": SubRecord({
+            "version": String(doc="Version of mongodb server"),
+            "git_version": String(doc="Git Version of mongodb server"),
+            "max_wire_version": Signed32BitInteger(),
+            "build_environment": SubRecord({
```

Comment:
Each of these subkeys should be a `WhitespaceAnalyzedStrings`, so that we don't tokenize at `/`, `-`, etc.

Reply:
Got it, thanks

Reply:
Actually, just within build_environment? or version / git_version too?

Reply:
note IRL - versions too